prov/efa/src/efa_user_info.c

```diff
@@ -354,6 +355,35 @@ int efa_user_info_alter_rxr(struct fi_info *info, const struct fi_info *hints)
 
 			info->domain_attr->mr_mode |= FI_MR_HMEM;
 
+			support_atomic = true;
+			if (hmem_ops[FI_HMEM_CUDA].initialized == true && cuda_get_xfer_setting() == false) {
```

Comment:
Can you add a todo here to loop over all interfaces, and if any one we cannot support atomics, we set `support_atomics` to false.

Reply:
done